release.json

```diff
@@ -13,6 +13,20 @@
 		"JMXFETCH_VERSION": "0.33.1",
 		"JMXFETCH_HASH": "140dd8568b78f5a616da8664f34bb37397c3d3ed87af106d632714f7218e8e88"
 	},
+	"7.16.1~rc.1": {
```

Comment:
I think we usually use dashes here instead of tildes, but everything should still work with this.

Reply:
pushed a commit, consistency is everything. Thanks!